src/main/java/io/cdap/plugin/gcp/spanner/sink/SpannerSink.java

```diff
@@ -183,16 +190,40 @@ private boolean isTablePresent(DatabaseClient dbClient) {
   }
 
   private Database getOrCreateDatabase(
+    BatchSinkContext context,
     DatabaseAdminClient dbAdminClient) throws ExecutionException, InterruptedException {
     Database database = getDatabaseIfPresent(dbAdminClient);
 
     if (database == null) {
       LOG.debug("Database not found. Creating database {} in instance {}.", config.getDatabase(), config.getInstance());
       // Create database
+      String cmekKey = context.getArguments().get(GCPUtils.CMEK_KEY);
+      if (cmekKey == null) {
+        database = dbAdminClient
+        .newDatabaseBuilder(DatabaseId.of(config.getProject(), config.getInstance(), config.getDatabase()))
```

Comment:
nit: indentation should be 2 more spaces

Reply:
Fixed.